common/src/main/java/org/apache/drill/common/exceptions/UserException.java

```diff
@@ -534,6 +535,19 @@ public UserException build(final Logger logger) {
 
       return newException;
     }
+
+    /**
+     * builds a user exception or returns the wrapped one. If the error is a system error, the error message is logged
+     * to the given {@link Logger}.
```

Comment:
you should probably update this comment

*the error message is logged to the UserException's logger*